test/testE2E.t.sol

```diff
@@ -131,7 +139,9 @@ contract E2ETests is Test {
     }
 
     function testMultipleDisputesDifferentChains() public {
-        // test multiple disputes on different parachains
+        // Test multiple disputes on different parachains
+        // Open disputes for identical values (same value, query id, & submission timestamp) on different parachains.
+        // Ensure no cross contamination in gov/staking contracts.
```

Comment:
re: cross contamination, it might be good to check that some of the storage variables are updated as expected, such as the identifierHash and paraId